tutorial/local-setup.md

```diff
@@ -4,27 +4,33 @@ This appendix describes how to check out, build and view the PHP documentation l
 Viewing results as a php.net mirror isn't a simple process, but it can be done.
 The following is one route, and it assumes:
 
-- PHP 5.4+ is available
+- PHP 7.2+ is available
 - Git version control system is available
+- DOM, libXML2, XMLReader, and SQLite3 are available
```

Comment:
Nit: I'd probably stick these with the PHP version, e.g. `PHP 7.2+ with the following extensions enabled: ...`